blackjax/mcmc/rmh.py

```diff
@@ -74,35 +73,6 @@ def init(position: PyTree, logdensity_fn: Callable) -> RMHState:
     return RMHState(position, logdensity_fn(position))
 
 
-def kernel():
-    """Build a Random Walk Rosenbluth-Metropolis-Hastings kernel with a gaussian
-    proposal distribution.
-
-    Returns
-    -------
-    A kernel that takes a rng_key and a Pytree that contains the current state
-    of the chain and that returns a new state of the chain along with
-    information about the transition.
-
-    """
-
-    def one_step(
-        rng_key: PRNGKey, state: RMHState, logdensity_fn: Callable, sigma: Array
-    ) -> Tuple[RMHState, RMHInfo]:
-
-        move_proposal_generator = normal(sigma)
-
-        def proposal_generator(key_proposal, position):
```

Comment:
`proposal_generator` builds a trajectory (`trajectory.py`) with an integrator (`integrators.py`, in this case the integrator is a proposal conditional on the previous position) to then create a proposal (`proposal.py`).

Reply:
Pretty sure the "trajectory" is given always by the user.